README.md

```diff
@@ -1,2 +1,34 @@
 # docker-spark-gpg
-Docker Alpine image with Spark, Python, Java and gpg. Used for local dev environment.
+Docker Alpine image with Spark, Python, Java and gpg. Used for loacal dev environment.
+
+Instructions
+
+
+1. Run the command in order to mount the export-data folder , the TestData folder and the gpg folder on to the root of the Docker container
+
+'docker run -v /Users/yasserkhan/Desktop/docker-spark-gpg/export-data:/root/export-data -v /Users/yasserkhan/Desktop/docker-spark-gpg/Spark:/root/Spark -v /Users/yasserkhan/Desktop/docker-spark-gpg/gpg:/root/gpg -it docker-spark-gpg :latest bash'
+
+
+NOTE: The gpg keys in the mounted directory will have already been imported. Use the command 'gpg --list-keys' to see a list of all keys imported. There will be a total of 4
```

Comment:
can you remove `There will be a total of 4` as that entirely depends on how many are in the mounted dir